tests/test_document.py

```diff
@@ -127,70 +138,45 @@ def test_get_by_uri(self):
         assert doc
         assert_equal(doc['title'], "document1") 
 
-    def test_get_all_by_uri(self):
+    @staticmethod
+    def test_get_all_by_uri():
```

Comment:
I think this test should test the public interface. In other words, make sure we get one document with all the links, rather than two dividend.